modules/apps/object/object-api/src/main/java/com/liferay/object/exception/RequiredObjectFieldRelationshipException.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.exception;
+
+import com.liferay.object.model.ObjectDefinition;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.exception.PortalException;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Larissa Ribeiro
+ */
+public class RequiredObjectFieldRelationshipException extends PortalException {
```

Comment:
@larissacribeiro to create a new exception, we must first update the [service.xml](https://github.com/carolmariaabb/liferay-portal/blob/bde9eb760be3e33c3037364b8ecde153eb6678bc/modules/apps/object/object-service/service.xml#L966-L1040) file and then run `gw buildService` to automatically generate this class.

However, this exception is not following the Liferay pattern, it should be `Required` + entityName, as you can see [here](https://github.com/carolmariaabb/liferay-portal/blob/bde9eb760be3e33c3037364b8ecde153eb6678bc/modules/apps/object/object-service/service.xml#L1037-L1039).

Please use the already existing `RequiredObjectFieldException` for this by creating a inner class like in [here](https://github.com/carolmariaabb/liferay-portal/blob/bde9eb760be3e33c3037364b8ecde153eb6678bc/modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectActionNameException.java#L26-L36).

Reply:
@carolmariaabb sorry about that, I've had never created one, so I wasn't aware of the process, so thanks for the thorough explanation, I've learned in case I need it again in the future! I've just updated the code, can you take another look? I'm unsure about what's the pattern in naming the classes.